src/generated/Normalizer/PriceNormalizer.php

```diff
@@ -52,6 +52,16 @@ public function denormalize($data, $class, $format = null, array $context = arra
         elseif (property_exists($data, 'value') && $data->{'value'} === null) {
             $object->setValue(null);
         }
+        if (property_exists($data, 'appliedPromotions') && $data->{'appliedPromotions'} !== null) {
+            $values = array();
+            foreach ($data->{'appliedPromotions'} as $value) {
+                $values[] = $value;
+            }
+            $object->setAppliedPromotions($values);
+        }
+        elseif (property_exists($data, 'appliedPromotions') && $data->{'appliedPromotions'} === null) {
+            $object->setAppliedPromotions(null);
+        }
```

Comment:
```suggestion
        if (property_exists($data, 'appliedPromotions')) {
            if ($data->{'appliedPromotions'} !== null) {
                $values = array();
                foreach ($data->{'appliedPromotions'} as $value) {
                    $values[] = $value;
                }
                $object->setAppliedPromotions($values);
            } else {
                $object->setAppliedPromotions(null);
            }
        }
```
This avoids checking the `property_exists` condition twice.
OIn addition, perhaps it is possible to avoid the `else` entirely if one always sets the applies promotions to null before the inner if(). I didn't do it in the suggestion simply because I'm not sure if that has any side effect.
Side note: do we still have to support ancient PHP versions that do not support the [] array syntax? If the answer is "no", then I'd replace the `$values = array();` line with `$values = [];`

Reply:
This file is automatically generated by jane, I would not change it